README.md

```diff
@@ -32,19 +32,18 @@ fastify.register(require('@fastify/mongodb'), {
   url: 'mongodb://mongo/mydb'
 })
 
-fastify.get('/user/:id', function (req, reply) {
+fastify.get('/user/:id', async function (req, reply) {
   // Or this.mongo.client.db('mydb').collection('users')
   const users = this.mongo.db.collection('users')
 
   // if the id is an ObjectId format, you need to create a new ObjectId
   const id = this.mongo.ObjectId(req.params.id)
-  users.findOne({ id }, (err, user) => {
-    if (err) {
-      reply.send(err)
-      return
-    }
+  try {
+    const user = await users.findOne({ id })
```

Comment:
```suggestion
    const user = await users.findOne({ id })
    return user
```

Reply:
In that case, we could also remove reply.send(user), right?